src/components/mode-tools/mode-tools.jsx

```diff
@@ -151,18 +157,29 @@ const ModeToolsComponent = props => {
     case Modes.RESHAPE:
         return (
             <div className={classNames(props.className, styles.modeTools)}>
-                <LabeledIconButton
-                    disabled={!props.hasSelectedUncurvedPoints}
-                    imgSrc={curvedPointIcon}
-                    title={props.intl.formatMessage(messages.curved)}
-                    onClick={props.onCurvePoints}
-                />
-                <LabeledIconButton
-                    disabled={!props.hasSelectedUnpointedPoints}
-                    imgSrc={straightPointIcon}
-                    title={props.intl.formatMessage(messages.pointed)}
-                    onClick={props.onPointPoints}
-                />
+                <InputGroup className={classNames(styles.modDashedBorder, styles.modLabeledIconHeight)}>
+                    <LabeledIconButton
+                        disabled={!props.hasSelectedUncurvedPoints}
+                        imgSrc={curvedPointIcon}
+                        title={props.intl.formatMessage(messages.curved)}
+                        onClick={props.onCurvePoints}
+                    />
+                    <LabeledIconButton
+                        disabled={!props.hasSelectedUnpointedPoints}
+                        imgSrc={straightPointIcon}
+                        title={props.intl.formatMessage(messages.pointed)}
+                        onClick={props.onPointPoints}
+                    />
+                </InputGroup>
+                <InputGroup className={classNames(styles.modLabeledIconHeight)}>
+                    <LabeledIconButton
+                        disabled={!(props.hasSelectedUncurvedPoints ||
```

Comment:
`!props.selectedItems.length` should suffice, an item is selected if any of its points are selected